unicodetools/src/main/java/org/unicode/text/utility/Settings.java

```diff
@@ -258,23 +258,24 @@ public String versionToString(VersionInfo version) {
 
     public static final class Output {
         /** The root of where we write output files. Most go into a "Generated" sub-folder. */
-        public static final String UNICODETOOLS_OUTPUT_DIR =
-                getRequiredPathAndFix("UNICODETOOLS_OUTPUT_DIR");
-        public static final String GEN_DIR = UNICODETOOLS_OUTPUT_DIR + "Generated/";
+        public static final String GEN_DIR =
+                getRequiredPathAndFix("UNICODETOOLS_GEN_DIR");
         public static final String BIN_DIR = GEN_DIR + "BIN/";
         public static final String GEN_UCD_DIR = GEN_DIR + "UCD/";
         public static final String GEN_UCA_DIR = GEN_DIR + "UCA/";
+        public static final String DATA_DIR = GEN_DIR + "DATA/";
```

Comment:
This could be a little confusing, because there is also a Settings.UnicodeTools.DATA_DIR.

It looks like all of these tools currently also write into a subfolder of whatever..._DIR they start from -- so why not just remove this line here, and change the Output.DATA_DIR in your current version to simply Output.GEN_DIR?

Reply:
Done.